src/lib/mongoBackend/MongoGlobal.h

```diff
@@ -150,6 +150,14 @@ using namespace mongo;
 
 /* ****************************************************************************
 *
+* OtisTreatFunction - callback signature for treatOntimeintervalSubscriptions
+*/
+typedef void (*OtisTreatFunction)(std::string tenant, BSONObj& bobj);
```

Comment:
Maybe `OtisTreatFunction` name is obsolete (as `OTI` stand for OnTimeInterval) and we should found a more general one...